### DISCUSSION TITLE
Relation is present in Admin endpoint, but not in Store endpoint

### DISCUSSION THREAD

**Discussion by AlexTechNoir** [medusajs/medusa#3801] [Q&A] [answered]
Good day!

In Store, when I query the `collections/{id}` endpoint with `expand` parameter, the products relation is not present in the response

![00](https://user-images.githubusercontent.com/43852391/231170338-99d0b795-e153-459b-a4fa-ed573250e20d.jpg)

If I fetch from `admin` endpoint, they _are_ present, it's like Admin and Store are not in sync

What am I missing here?

upd: same case for searching products in POST request `http://localhost:9000/store/products/search?q=Shirt` - there are no matches even when searching based on one letter, like there are no products at all

**Comment by olivermrbl** [comment, marked as answer]
@AlexTechNoir – I recommend using the filters on the list endpoint to retrieve products in specific product categories e.g. `GET /store/products?category_id[]=pcat_123456789`. You can even pass several ids to the endpoint `GET /store/products?category_id[]=pcat_123456789&category_id[]=pcat_987654321`.

Regarding the search endpoint, if you don't have a [search plugin](https://docs.medusajs.com/plugins/search) installed, you won't be able to use this endpoint. Though, you can use the same list endpoint as before, to achieve your goal: `GET /store/products?q=Shirt&limit=5`.

I hope this helps. I will highlight your first issue for our team.

**Reply by AlexTechNoir** [reply]
Yes, that's what I ended up doing, thank you!
